.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -12,14 +12,14 @@ jobs:
   release:
     strategy:
       matrix:
-        python-version: [ 3.8 ]
+        python-version: [ 3.10 ]
```

Comment:
When specifying versions like Python `3.10` in GitHub workflows, I recommend to use strings `"3.10"`. Rationale: I had cases in which yaml probably interpreted the `3.10` as floating point and did cut off the trailing zero resulting in `3.1` which made the build fail.